src/Core/Rexl.Bind/Operations/Funcs/Text.cs

```diff
@@ -758,3 +758,66 @@ public static string Exec(string src, string remove, string insert)
         return src.Replace(remove, insert);
     }
 }
+
+public sealed partial class TextPadFunc : RexlOper
```

Comment:
It would be good to add a doc comment using `///` explaining a bit what the functions do. We don't have these comments on many functions but we really should.